sympy/core/tests/test_args.py

```diff
@@ -590,6 +590,22 @@ def test_sympy__calculus__util__AccumulationBounds():
     from sympy.calculus.util import AccumulationBounds
     assert _test_args(AccumulationBounds(0, 1))
 
+@XFAIL
+def test_sympy__sets__ordinals__OmegaPower():
+    from sympy.sets.ordinals import OmegaPower
+    assert _test_args(OmegaPower(2,))
```

Comment:
I think that there should be acceptable arguments.

Reply:
sorry, this was a typo.